### DISCUSSION TITLE
Can this be considered as a update/contnue of developement of official MeGUI, or rather an independent fork?

### DISCUSSION THREAD

**Discussion by obiwan-bartek** [Kurtnoise-zeus/megui#27] [General]
Hi,

I believe there is still a relatively large group of people using MeGUI, and as we all know Zathor, the original author of MeGUI does not update MeGUI for some time. Maybe we should all group together and continue his work there (or in any other place), to keep MeGUI (both x86 and x64) keep and running?

**Comment by Kurtnoise-zeus** [comment]
Hi,

I used to work within Zathor on the official MeGUI several years ago but unfortunately I lost my sourceforge account and cannot login anymore on MeGUI project. Thats why I decided to fork it through there.

I tried to contact Zathor a couple days ago but he did not reply yet.

Basically, yes, Im trying to continue his work there (but only on x64 area).

**Comment by obiwan-bartek** [comment]
Oh I see. What a pity. Is there no way to make a new sourceforge account and add it to MeGUI owners/authors? Wouldn't it be more convenient?

**Comment by Kurtnoise-zeus** [comment]
I could create a new account on Sourceforge, yes, but Zathor beeing missing in action, I couldnt be added to the project at the moment.
